monai/config/deviceconfig.py

```diff
@@ -260,5 +262,14 @@ def print_debug_info(file: TextIO = sys.stdout) -> None:
     print_gpu_info(file)
 
 
+@deprecated(since=1.4, removed=1.6, msg_suffix="Please use `monai.utils.enums.IgniteInfo` instead.")
+class IgniteInfo:
+    """
+    IgniteInfo enum is moved to `monai.utils.enums.IgniteInfo`.
```

Comment:
We can condense these three docstring lines into one:

```
"""Deprecated Import of IgniteInfo enum, which was moved to `monai.utils.enums.IgniteInfo`."""
```

Reply:
Updated, thx!